charts/pip-data-management/values.dev.template.yaml

```diff
@@ -8,7 +8,9 @@ java:
     DB_USER: "{{ .Values.postgresql.postgresqlUsername}}"
     DB_PASS: "{{ .Values.postgresql.postgresqlPassword}}"
     DB_PORT: "5432"
+    #AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY: '3c3bacb7-64e0-476c-aeaa-657657b063fd'
```

Comment:
should this be in here?

Reply:
It has been left in as an example on how to get it working in a Pull Request, if we need to come back to it.

Reply:
got it 👍🏼